src/cursor.rs

```diff
@@ -36,7 +36,7 @@ use wire_protocol::operations::Message;
 
 use std::collections::vec_deque::VecDeque;
 
-pub const DEFAULT_BATCH_SIZE: i32 = 20;
+pub const DEFAULT_BATCH_SIZE: i32 = 0;
```

Comment:
Let's add a little comment here about this allowing the server to decide the batch size?